### PR TITLE
[NDS-338] Feat/Props Standardization (Booleans pt4) 

### PR DESCRIPTION
**Pull request by Christos-Zacharopoulos**
## Description

To be merged after [this](https://github.com/Orfium/orfium-ictinus/pull/520)

**FOR V5**

This PR is part of a chain of PRs to follow regarding props globalization.

ticket: https://orfium.atlassian.net/browse/NDS-338

## Test Plan

Everything will be updated on a PR later. It's too much to keep track right now.

### CODE REVIEW

File: src/components/Drawer/Drawer.tsx [hunk 1]
```diff
@@ -9,7 +8,7 @@ import { MenuItem } from './types';
 
 export type Props = {
   /** Defines if the drawer is expanded */
-  expanded: boolean;
+  isExpanded: boolean;
   /** Changes if the drawer is expanded */
   setExpanded: (v: boolean) => void;
```
Comment (panvourtsis) [inline src/components/Drawer/Drawer.tsx:13]: 🤔 the is `setIsExpanded` right?

File: src/components/Drawer/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -12,18 +12,18 @@ const Navigation: React.FC<NavigationProps> = ({ menuItems, expanded }) => {
...
-    setOpenMenuItems(openMenuItems => (openMenuItems.indexOf(newUrl) !== -1 ? [] : [newUrl]));
+    setOpenMenuItems((openMenuItems) => (openMenuItems.indexOf(newUrl) !== -1 ? [] : [newUrl]));
   }, []);
 
   return (
     <div css={navigationContainerStyle(expanded)}>
       {menuItems.map(
-        menuItem =>
+        (menuItem) =>
           menuItem.visible && (
```
Comment (panvourtsis) [inline src/components/Drawer/Navigation/Navigation.tsx:22]: same with `visible`

**Review by panvourtsis — APPROVED** [review]
I am approving this because the two comments that refer to code outside of the context might be fixed in other PR so let me know

**Comment by Christos-Zacharopoulos** [comment]
@panvourtsis Both these issues are solved in https://github.com/Orfium/orfium-ictinus/pull/524 and https://github.com/Orfium/orfium-ictinus/pull/525

**Pull request merged**